engine/userguide/networking/index.md

```diff
@@ -400,6 +400,20 @@ A bridge network is useful in cases where you want to run a relatively small
 network on a single host. You can, however, create significantly larger networks
 by creating an `overlay` network.
 
+### The `docker_gwbridge` network
+
+The `docker_gwbridge` network is a local bridge network which is automatically created by Docker when
+you join the local Docker daemon to a swarm. It facilitates networks using the `overlay` network driver.
+When none of a container's available networks can provide external connectivity, Docker automatically
+connects the container to the `docker_gwbridge` network in addition to its other networks, so the
+container can connect to external hosts. 
+
+Docker creates the `docker_gwbridge` network automatically. If you need to customize the network, such as
+customizing its subnet, you can create it manually before it is needed. The following is an example command.
+
+```bash
+docker network create --subnet <...> --opt com.docker.network.bridge.name=docker_gwbridge --opt com.docker.network.bridge.enable_icc=false docker_gwbridge
```

Comment:
Some minor nits;

- missing a `$` prompt here
- maybe we should add an actual value for `--subnet` instead of the `<...>` placeholder
- for readability, we should wrap this example